pkg/kubectl/cmd/create_secret.go

```diff
@@ -116,25 +116,27 @@ func CreateSecretGeneric(f cmdutil.Factory, cmdOut io.Writer, cmd *cobra.Command
 }
 
 var (
-	secretForDockerRegistryLong = templates.LongDesc(`
-		Create a new secret for use with Docker registries.
+	secretForDockerRegistryLong = templates.LongDesc(i18n.T(`
```

Comment:
@brendandburns looks like the description and examples from generic secrets are being used in the tls and docker registry subcommands here, maybe the script is not safe for files with more than one description and examples?  ;)